CMakeLists.txt

```diff
@@ -34,58 +31,52 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
         execute_process(COMMAND ${CMAKE_CXX_COMPILER} -dumpversion OUTPUT_VARIABLE CMAKE_CXX_COMPILER_VERSION)
     endif ()
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.0)
-        message("CXX COMPILER VERSION: ${CMAKE_CXX_COMPILER_ID} : ${CMAKE_CXX_COMPILER_VERSION}")
-        message(FATAL_ERROR "Gadgetron requires GCC version >= 6.0")
+        message(STATUS "CXX COMPILER VERSION: ${CMAKE_CXX_COMPILER_ID} : ${CMAKE_CXX_COMPILER_VERSION}")
+        message(FATAL_ERROR "Pingvin requires GCC version >= 6.0")
     endif ()
 elseif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.5)
-        message(FATAL_ERROR "Gadgetron requires Clang version >= 3.5")
+        message(FATAL_ERROR "Pingvin requires Clang version >= 3.5")
     endif ()
     add_compile_definitions(BOOST_NO_CXX98_FUNCTION_BASE)
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 18.0)
-        message(FATAL_ERROR "Gadgetron requires MSVC 2013 or above ")
+        message(FATAL_ERROR "Pingvin requires MSVC 2013 or above ")
     endif ()
 else ()
     message(WARNING "Unsupported compiler!")
 endif ()
 
 #VERSIONING AND INSTALL PATHS
-set(GADGETRON_VERSION_MAJOR 4)
-set(GADGETRON_VERSION_MINOR 7)
-set(GADGETRON_VERSION_PATCH 2)
-set(GADGETRON_VERSION_STRING ${GADGETRON_VERSION_MAJOR}.${GADGETRON_VERSION_MINOR}.${GADGETRON_VERSION_PATCH})
-set(GADGETRON_SOVERSION ${GADGETRON_VERSION_MAJOR}.${GADGETRON_VERSION_MINOR})
+set(PINGVIN_VERSION_MAJOR 5)
```

Comment:
I guess this version choice bases us off Gadgetron? Could also be 0.1.0? We work towards Pingvin 1.0?